crates/context_aware_config/migrations/2024-11-05-083200_add_priority_restructure/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+ALTER TABLE public.dimensions
+add column position integer DEFAULT 1 NOT NULL;
+
+ALTER TABLE public.contexts
```

Comment:
Can we write a query to copy priority to weightage

Reply:
@Datron we can write , but we will anyway have to re populate dimension position and then recompute

migration approach: https://github.com/juspay/superposition/discussions/277#discussioncomment-11318312